src/inmanta/app.py

```diff
@@ -502,6 +506,11 @@ def cmd_parser():
     return parser
 
 
+def get_current_version_and_exit():
+    print("Current Inmanta version:", get_compiler_version())
```

Comment:
Minor: We try to use f-strings as much as possible is we need to parameterize a string. This improves readability and keeps code style of our project more consistent.

Reply:
Thanks for the notice, sure thing.
Is there a way to provide some warning about it and the other conventions as well (that are not covered by pep8)? Or it's not worth the effort (right now)?